R/list.common.R

```diff
@@ -16,8 +16,12 @@
 #' foo <- list(x = LETTERS[1:3], y = LETTERS[3:5])
 #' list.common(foo)
 list.common <- function(.data, expr) {
-  if (length(.data) == 0L) return(NULL)
-  expr <- if(missing(expr)) quote(.) else substitute(expr)
+
+  if (!length(.data)) {
+    return(NULL)
+  }
+
+  expr <- ifelse(missing(expr), quote(.), substitute(expr))
```

Comment:
The use of `ifelse` here breaks the [example](https://travis-ci.org/renkun-ken/rlist/jobs/83261755#L2520). `ifelse` only works for vector objects but not for language objects.

For example,

```r
ifelse(FALSE, quote(.), substitute(names(score)))
```

will produce an error 

```
Error in rep(no, length.out = length(ans)) : 
  attempt to replicate an object of type 'language'
```